workflow/rules/purecn.smk

```diff
@@ -0,0 +1,112 @@
+__author__ = "Erik Demitz-Helin"
+__copyright__ = "Copyright 2022, Erik Demitz-Helin"
+__email__ = "erik.demitz-helin@gu.se"
+__license__ = "GPL-3"
+
+
+rule purecn_coverage:
+    input:
+        bam="alignment/samtools_merge_bam/{sample}_{type}.bam",
+        bai="alignment/samtools_merge_bam/{sample}_{type}.bam.bai",
+    output:
+        temp(
+            expand(
+                "cnv_sv/purecn_coverage/{{sample}}_{{type}}{ext}",
+                ext=[
+                    "_coverage.txt.gz",
+                    "_coverage_loess.txt.gz",
+                    "_coverage_loess.png",
+                    "_coverage_loess_qc.txt",
+                ],
+            )
+        ),
+    params:
+        intervals=config.get("purecn_coverage", {}).get("intervals", ""),
+        extra=config.get("purecn_coverage", {}).get("extra", ""),
+    log:
+        "cnv_sv/purecn_coverage/{sample}_{type}.output.log",
+    benchmark:
+        repeat(
+            "cnv_sv/purecn_coverage/{sample}_{type}.output.benchmark.tsv",
+            config.get("purecn_coverage", {}).get("benchmark_repeats", 1),
+        )
+    threads: config.get("purecn_coverage", {}).get("threads", config["default_resources"]["threads"])
+    resources:
+        mem_mb=config.get("purecn_coverage", {}).get("mem_mb", config["default_resources"]["mem_mb"]),
+        mem_per_cpu=config.get("purecn_coverage", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("purecn_coverage", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("purecn_coverage", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("purecn_coverage", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("purecn_coverage", {}).get("container", config["default_container"])
+    conda:
+        "../envs/purecn_coverage.yaml"
```

Comment:
```suggestion
        "../envs/purecn.yaml"
```

we should try to use one cond env file for the same tool, and not one new for each rule